README.md

```diff
@@ -1,184 +1,71 @@
---[ DiscoveryAck{  } ]----------------------------------------------------------------->
-```
-
-## Dialing flow
-
-```
-Client A                                Server                                  Client B
-
-The client opens a muxed connection and sends the peerID of the target over that connection
---[ *opens connection* ]-------------->
---[ DialRequest{ target: <bytes> } ]-->
-
-The server verifies if the client is online and responds with either an error or opens a connection to the other and forwards that over the existing connection
-                                        --[ *opens connection* ]--------------->
-<-[ DialRequest{ error? } ]------------
----------------------------------------->-[ *forwarded connection* ]----------->
-
-After that the normal libp2p dialing flow is happening between A and B
-```
-
-# Choices and explanations
-
-## Why not use libp2p's built-in switch instead of microswitch?
-
-- Complexity: Dynamic injection isn't a thing yet, and any other solution would be a complete mess
-- Performance: As stardust will mainly be used over wss:// there is no need to add another layer of SECIO on top of that (connections between peers are verified and protected by SECIO anyways)
-
-## Crypto challenge
-
-- Using a signature challenge would make the client sign anything the server gives it
-- Using a decryption challenge would make the client decrypt anything the server gives it
-- Using a hash-based decryption challenge solves those problems as the decrypted data is hashed together with random data so that even if the client were to be tricked into decrypting secrets, those would come back as an useless hash to the attacker
-
-## Not using PoW for registrations like rendezvous
-
-That's why it takes forever to be developed ;)
+> `libp2p-stardust` is one of the multiple transports available for libp2p. `libp2p-stardust` incorporates both a transport and a discovery service that is facilitated by the stardust server, also available in this repo.
 
 ## Lead Maintainer
 
 [Maciej Krüger](https://github.com/mkg20001)
 
-## Description
-
-`libp2p-stardust` is one of the multiple transports available for libp2p. `libp2p-stardust` incorporates both a transport and a discovery service that is facilitated by the stardust server, also available in this repo and module.
-
-## Todo
+## Usage
 
- - [ ] Integrate as libp2p transport (WIP)
- - [x] Add discovery
- - [ ] Split server into it's own repo
+### Install
 
-## Usage
+```bash
+> npm install libp2p-stardust
+```
 
 ### Example
 
 ```js
-'use strict'
-
-const Libp2p = require('libp2p')
-const Id = require('peer-id')
-const Info = require('peer-info')
+const Stardust = require('libp2p-stardust')
 const multiaddr = require('multiaddr')
-const pull = require('pull-stream')
+const pipe = require('it-pipe')
+const { collect } = require('streaming-iterables')
 
-const Stardust = require('libp2p-stardust')
+const addr = multiaddr('/ip4/188.166.203.82/tcp/5892/ws/p2p-stardust')
```

Comment:
shouldn't we give this a proper dnsaddr? or is this just a tmp addr?

Reply:
yeah, after this PR being merged, we will need to deploy the new version of the server, and by then we can update this

Reply:
tracking here https://github.com/libp2p/js-libp2p-stardust/issues/15

Reply:
Thanks ❤️